Zilon.Core/Zilon.Core.Tests/StaticObjectModules/TestCaseDataSources/DepositDurabilityModuleTestCaseDataSource.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+
+using NUnit.Framework;
+
+namespace Zilon.Core.Tests.StaticObjectModules.TestCaseDataSources
+{
+    public static class DepositDurabilityModuleTestCaseDataSource
+    {
+        public static IEnumerable DestroyTestCases
+        {
+            get
+            {
+                // StockValue = 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5427118)